src/azure/Templates/AzureMethodGroupTemplateAsync.cshtml

```diff
@@ -9,6 +9,11 @@
 @Header("# ").TrimMultilineHeader()
 # --------------------------------------------------------------------------
 @EmptyLine
+@if (Model.AddTracingDecorators)
+{
+@:from azure.core.tracing.decorator import distributed_trace
```

Comment:
This is the async file, do you need this?

Reply:
taken out

Reply:
actually paging has sync functions even in async packages.